fleet_management_api/logs.py

```diff
@@ -0,0 +1,53 @@
+from __future__ import annotations
+import json
+import os
+import logging.config
+
+from typing import TypeVar, Mapping
+
+
+T = TypeVar("T", bound=Mapping)
+
+
+LOG_FILE_NAME = "fleet_management_http_api.log"
+
+
+def clear_logs() -> None:
+    """Clear the log file."""
+    with open(f"log/{LOG_FILE_NAME}", "w") as f:
+        f.write("")
+
+
+def configure_logging(config_path: str) -> None:
+    try:
+        with open(config_path) as f:
+            logging.config.dictConfig(json.load(f))
+    except Exception:
+        logger = logging.getLogger("werkzeug")
+        logger.warning(
+            f"Fleet Protocol HTTP API: Could not find a logging configuration file (entered path: {config_path}. Using default logging configuration."
+        )
+        if not os.path.isfile(f"log/{LOG_FILE_NAME}"):
+            if not os.path.exists("log"):
+                os.makedirs("log")
+            with open(f"log/{LOG_FILE_NAME}", "w") as f:
+                f.write("")
+
+        logger.propagate = False
+        formatter = logging.Formatter(
+            fmt="[%(asctime)s.%(msecs)03d] [fleet-management-http-api] [%(levelname)s]\t %(message)s",
```

Comment:
Isn't this set in the config?

Reply:
In case of missing config, this is applied instead